2021-01-31 11:11:24.943 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package azureml-train-automl missing from dependencies file.
2021-01-31 11:11:24.943 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: azureml-train-automl==1.20.0.*
2021-01-31 11:11:24.944 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package pandas missing from dependencies file.
2021-01-31 11:11:24.944 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: pandas==0.25.1
2021-01-31 11:11:24.945 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package psutil missing from dependencies file.
2021-01-31 11:11:24.945 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: psutil>5.0.0,<6.0.0
2021-01-31 11:11:24.945 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package scikit-learn missing from dependencies file.
2021-01-31 11:11:24.946 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: scikit-learn==0.22.1
2021-01-31 11:11:24.946 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package numpy missing from dependencies file.
2021-01-31 11:11:24.946 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: numpy~=1.18.0
2021-01-31 11:11:24.946 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package py-xgboost missing from dependencies file.
2021-01-31 11:11:24.947 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: py-xgboost<=0.90
2021-01-31 11:11:24.948 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package inference-schema missing from dependencies file.
2021-01-31 11:11:24.949 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package fbprophet missing from dependencies file.
2021-01-31 11:11:24.949 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: fbprophet==0.5
2021-01-31 11:11:24.949 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package setuptools-git missing from dependencies file.
2021-01-31 11:11:24.949 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:209 - Using installed version: setuptools-git==1.2
2021-01-31 11:11:24.964 - INFO - 5411 - azureml.train.automl._azureautomlclient._validate_input:664 - Start data validation.
2021-01-31 11:11:28.065 - CRITICAL - 5411 - azureml.train.automl._azureautomlclient.log_traceback:224 - Type: ExecutionFailure
Class: ValidationException
Message: ValidationException:
	Message: Failed to execute the requested operation: data/settings validation. Error details: Validation error(s): [{
    "additional_properties": {
        "debugInfo": null
    },
    "code": "UserError",
    "severity": 2,
    "message": "Y is empty",
    "message_format": "{dataName} is empty",
    "message_parameters": {
        "dataName": "Y"
    },
    "reference_code": null,
    "details_uri": null,
    "target": "training_data",
    "details": [
        {
            "additional_properties": {
                "debugInfo": null
            },
            "code": null,
            "severity": null,
            "message": "null",
            "message_format": null,
            "message_parameters": {},
            "reference_code": null,
            "details_uri": null,
            "target": null,
            "details": [],
            "inner_error": null
        }
    ],
    "inner_error": {
        "additional_properties": {},
        "code": "BadData",
        "inner_error": {
            "additional_properties": {},
            "code": "EmptyData",
            "inner_error": null
        }
    }
}]
	InnerException: None
	ErrorResponse 
{
    "error": {
        "code": "UserError",
        "message": "Failed to execute the requested operation: data/settings validation. Error details: Validation error(s): [{\n    \"additional_properties\": {\n        \"debugInfo\": null\n    },\n    \"code\": \"UserError\",\n    \"severity\": 2,\n    \"message\": \"Y is empty\",\n    \"message_format\": \"{dataName} is empty\",\n    \"message_parameters\": {\n        \"dataName\": \"Y\"\n    },\n    \"reference_code\": null,\n    \"details_uri\": null,\n    \"target\": \"training_data\",\n    \"details\": [\n        {\n            \"additional_properties\": {\n                \"debugInfo\": null\n            },\n            \"code\": null,\n            \"severity\": null,\n            \"message\": \"null\",\n            \"message_format\": null,\n            \"message_parameters\": {},\n            \"reference_code\": null,\n            \"details_uri\": null,\n            \"target\": null,\n            \"details\": [],\n            \"inner_error\": null\n        }\n    ],\n    \"inner_error\": {\n        \"additional_properties\": {},\n        \"code\": \"BadData\",\n        \"inner_error\": {\n            \"additional_properties\": {},\n            \"code\": \"EmptyData\",\n            \"inner_error\": null\n        }\n    }\n}]",
        "inner_error": {
            "code": "ExecutionFailure"
        }
    }
}
Traceback:
  File "_azureautomlclient.py", line 410, in fit
    test_data=test_data)
  File "_azureautomlclient.py", line 424, in _fit_remote
    validation_data=validation_data, test_data=test_data)
  File "_azureautomlclient.py", line 487, in _fit_remote_core
    training_data=training_data, validation_data=validation_data, test_data=test_data)
  File "_azureautomlclient.py", line 538, in _create_parent_run_for_remote
    test_data=test_data
  File "_azureautomlclient.py", line 617, in _create_and_validate_parent_run_dto
    self._validate_input(parent_run_dto)
  File "_azureautomlclient.py", line 694, in _validate_input
    ExecutionFailure, operation_name="data/settings validation", error_details=msg)

ExceptionTarget: Unspecified
2021-01-31 11:11:28.065 - INFO - 5411 - azureml.train.automl._azureautomlclient._fail_parent_run:847 - No parent run to fail
2021-01-31 11:12:15.473 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package azureml-train-automl missing from dependencies file.
2021-01-31 11:12:15.473 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: azureml-train-automl==1.20.0.*
2021-01-31 11:12:15.473 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package pandas missing from dependencies file.
2021-01-31 11:12:15.473 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: pandas==0.25.1
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package psutil missing from dependencies file.
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: psutil>5.0.0,<6.0.0
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package scikit-learn missing from dependencies file.
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: scikit-learn==0.22.1
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package numpy missing from dependencies file.
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: numpy~=1.18.0
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package py-xgboost missing from dependencies file.
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: py-xgboost<=0.90
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package inference-schema missing from dependencies file.
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package fbprophet missing from dependencies file.
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: fbprophet==0.5
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package setuptools-git missing from dependencies file.
2021-01-31 11:12:15.474 - INFO - 5411 - azureml.train.automl._azureautomlclient.modify_run_configuration:209 - Using installed version: setuptools-git==1.2
2021-01-31 11:12:15.480 - INFO - 5411 - azureml.train.automl._azureautomlclient._validate_input:664 - Start data validation.
2021-01-31 11:12:17.985 - INFO - 5411 - azureml.train.automl._azureautomlclient._validate_input:677 - Validation service found the data has no errors.
2021-01-31 11:12:17.985 - INFO - 5411 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:542 - Start creating parent run.
2021-01-31 11:12:18.574 - INFO - 5411 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:562 - [ParentRunID:AutoML_8132042c-c606-4d2c-ae6a-f50d70b741ca] Remote run using input X and y.
2021-01-31 11:12:18.821 - INFO - 5411 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:575 - Parent Run ID: AutoML_8132042c-c606-4d2c-ae6a-f50d70b741ca
2021-01-31 11:12:18.821 - INFO - 5411 - azureml.train.automl._azureautomlclient._fit_remote_core:503 - Starting a snapshot run (snapshot_id : None)
